### DISCUSSION TITLE
What is the best way to work with dynamic styles in react-native-web?

### DISCUSSION THREAD

**Discussion by RWOverdijk** [necolas/react-native-web#2032] [Q&A]
This might be a really weird question.

With StyleSheet.create() you get classnames that can be used to apply styles to components. That's pretty awesome.

But what if I no longer need a style? I'm talking specifically about dynamic styles that are based on user input. I will create new classes using StyleSheet.create(), but the old classes will still be there. Can those be removed somehow?

It's not a huge deal, I'm just curious if it can.

**Update:** I just figured, perhaps another approach could be a way to update stylesheets/use css variables? Most of the changes I am talking about are related to colors and such. Those are all variables. If a stylesheet could be updated then those styles could be applied.

**Comment by steida** [comment]
You can use CSS variables, but then your code will not be multiplatform.

**Reply by RWOverdijk** [reply]
True, but what would be the cross-platform solution then?

Also do you know if styleSheets can be cleaned up?

**Reply by steida** [reply]
Probably not and I think it's ok. For dynamic styles, just use them View style={{width: 111}} without StyleSheet.create. It will be rendered as inline styles and it's OK.
